packages/pluginutils/test/makeLegalIdentifier.ts

```diff
@@ -17,3 +17,7 @@ test('blacklists arguments (https://github.com/rollup/rollup/issues/871)', (t) =
 test('empty', (t) => {
   t.is(makeLegalIdentifier(''), '_');
 });
+
+test('handles input evaluated to blacklisted identifier', (t) => {
```

Comment:
this kinda mentions the internal implementation, but not sure how to avoid it as the bug was caused by the way this was implemented